app/controllers/v0/dependents_applications_controller.rb

```diff
@@ -23,7 +23,7 @@ def create
     def show
       dependent_service = BGS::DependentService.new
       response = dependent_service.get_dependents(current_user)
-      render json: response, each_serializer: DependentsSerializer
+      render json: { data: { attributes: response } }, each_serializer: DependentsSerializer
```

Comment:
Can you confirm that the `DependentsSerializer ` is being used at all? `response` is just a `Hash` and it looks like you mean to iterate over `response[:persons].

This observation came from @annaswims (many thanks!)

Reply:
https://dsva.slack.com/archives/CQHBJ5U06/p1582725724033300

Reply:
You were right...thank you @omgitsbillryan and @annaswims.  I've made the fix so now the serializer is being used correctly.  Please take a look when you get a chance and let me know if this is ok to merge.  Thanks!